pkg/cli/config/initialize/basecamp.go

```diff
@@ -119,6 +119,73 @@ type BaseCampGlobals struct {
 }
 
 // Basecamp Defaults
+// master bootcmd (cloud-init user-data)
+var masterBootCMD = [][]string{
+	{"cloud-init-per", "once", "create_PART_LVM", "parted", "--wipesignatures", "-m", "--align=opt", "--ignore-busy", "/dev/md/AUX", "mkpart", "primary", "1MiB", "100%"},
+	{"cloud-init-per", "once", "create_PV", "pvcreate", "-ff", "-y", "-M", "lvm2", "/dev/md/AUX1"},
+	{"cloud-init-per", "once", "create_VG", "vgcreate", "metalvg0", "/dev/md/AUX1"},
+	{"cloud-init-per", "once", "create_LV_CRAYS3CACHE", "lvcreate", "-l", "25%PVS", "-n", "CRAYS3CACHE", "-y", "metalvg0"},
+	{"cloud-init-per", "once", "create_LV_CONRUN", "lvcreate", "-l", "4%PVS", "-n", "CONRUN", "-y", "metalvg0"},
+	{"cloud-init-per", "once", "create_LV_CONLIB", "lvcreate", "-l", "36%PVS", "-n", "CONLIB", "-y", "metalvg0"},
+	{"cloud-init-per", "once", "create_LV_K8SLET", "lvcreate", "-l", "10%PVS", "-n", "K8SLET", "-y", "metalvg0"},
+}
+
+// master fs_setup (cloud-init user-data)
+var masterFileSystems = []map[string]interface{}{
+	{"label": "CRAYS3CACHE", "filesystem": "ext4", "device": "/dev/disk/by-id/dm-name-metalvg0-CRAYS3CACHE", "partition": "auto", "overwrite": true},
+	{"label": "CONRUN", "filesystem": "xfs", "device": "/dev/disk/by-id/dm-name-metalvg0-CONRUN", "partition": "auto", "overwrite": true},
+	{"label": "CONLIB", "filesystem": "xfs", "device": "/dev/disk/by-id/dm-name-metalvg0-CONLIB", "partition": "auto", "overwrite": true},
+	{"label": "K8SLET", "filesystem": "xfs", "device": "/dev/disk/by-id/dm-name-metalvg0-K8SLET", "partition": "auto", "overwrite": true},
```

Comment:
nit: put some of your commonly repeated strings like `metalvg0`, `CRAYS3CACHE`, `CONRUN`, `CONLIB`, and `K8SLET` as const/vars somewhere else in this file and use `fmt.Sprintf`.